modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js

```diff
@@ -310,7 +310,19 @@ export default withRouter(
 
 			let fn;
 
-			if (filter === 'latest-edited') {
+			if (filter === 'latest-created') {
+				fn = getThreadsCallback(
+					creatorId,
+					currentTag,
+					page,
+					pageSize,
+					search,
+					section,
+					siteKey,
+					'dateCreated:desc'
+				);
+			}
+			else if (filter === 'latest-edited') {
```

Comment:
Can you just add 'dateCreated:desc' as the default value in the last else call?

Reply:
I don't have a problem of making it explicit but I would remove the else block if there is already an explicit default.

Reply:
Sure! I doubt about it, but I was concern that we added more cases in the future that may interfere